src/functions/template-tags/link.php

```diff
@@ -250,15 +250,22 @@ function tribe_get_listview_dir_link( $direction = 'next', $term = null, $curren
 }
 
 /**
- * Utility function to update the pagination and current display on the list view.
+ * Updates pagination and determines the current view display for the list view.
+ *
+ * This function adjusts the pagination and sets the view to be displayed based on
+ * the current page and navigation direction.
  *
  * @since 4.6.12
  *
- * @param int $page
- * @param string $direction
- * @param null $currently_displaying
+ * @param int         $page                 The current page number. Default is 1.
+ * @param string      $direction            The navigation direction, either 'next' or 'prev'. Default is 'next'.
+ * @param string|null $currently_displaying The view currently being displayed, such as 'list' or 'past'.
+ *                                          Default is null, which triggers a fallback to the default list view.
  *
- * @return array
+ * @return array { An associative array containing the updated display and page number.
+ *      @type string $display The view to be displayed, either 'list' or 'past'.
+ *      @type int    $page    The updated page number.
+ * }
```

Comment:
I don't like this style BUT i am not sure what our docs say as well. Could you confirm if that's how its expected we style associative arrays in docblocks please ?

Reply:
I'm adjusting based on this documentation: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#other-tags